fairseq/options.py

```diff
@@ -225,7 +225,7 @@ def get_parser(desc, default_task="translation"):
     parser.add_argument('--cpu', action='store_true', help='use CPU instead of CUDA')
     parser.add_argument('--tpu', action='store_true', help='use TPU instead of CUDA')
     parser.add_argument('--bf16', action='store_true', help='use bfloat16; implies --tpu')
-    parser.add_argument('--fp16', action='store_true', help='use FP16')
```

Comment:
This was done in my previous commit:
https://github.com/SciPioneer/fairseq/commit/0fe2326ef7ffa50516085394f236e6a1e58fa48e